packages/webdriverio/tests/utils/Timer.test.js

```diff
@@ -0,0 +1,57 @@
+import Timer from '../../src/utils/Timer'
+// import sinon from 'sinon'
+import assert from 'assert'
+
+describe('timer', () => {
+    describe('promise', () => {
+        it('should be rejected by timeout', () => {
+            let timer = new Timer(20, 30, () => Promise.resolve(false))
+
+            return timer.then(assert.fail, (e) => {
```

Comment:
Let's use async/await syntax and jests assertion utilities

Reply:
Ok that makes sense.  I have a pretty busy day so probably won't get to this until tonight